container/service.go

```diff
@@ -94,7 +94,7 @@ func (c *DockerContainer) deletePendingContainer(namespace []string) error {
 	// This hack for Docker to stop and then remove the container.
 	// See issue https://github.com/moby/moby/issues/32620
 	if container.ContainerJSONBase != nil {
-		timeout := 1 * time.Second
+		timeout := 10 * time.Second // 10 second is the default value docker use
```

Comment:
```suggestion
		c.client.ContainerStop(ctx, container.ID, nil)
```

If you wan't the default value then pass nil to ContainerStop (this is why `time.Duration` is pointer here.

Link to docs - https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/client/container_stop.go#L15

Otherwise if you want to change docker default value what's the motivation behind?

Reply:
👍